apps/sensenet/cypress/integration/task-list/task-list.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { pathWithQueryParams } from '../../../src/services/query-string-builder'
+
+const taskName = `Test Task List`
+
+describe('Task-List', () => {
+  before(() => {
+    cy.login()
+    cy.visit(pathWithQueryParams({ path: '/', newParams: { repoUrl: Cypress.env('repoUrl') } }))
+  })
+  it('Task should be created', () => {
+    cy.get('[data-test="drawer-menu-item-Content"]').click()
+    cy.get('[data-test="menu-item-IT Workspace"]').click()
+    cy.get('[data-test="add-button"').click()
+    cy.get('[data-test="listitem-Task list"')
+      .click()
+      .then(() => {
+        cy.get('#DisplayName').type(taskName)
+        cy.get('#Name').type(taskName)
+        cy.contains('Submit').click()
+        cy.get(`[data-test="table-cell-${taskName}"]`).should('have.text', taskName)
```

Comment:
Steps 8 and 9 are missing

Reply:
step 8: done
step 9: There is no item "System Folder".
<img width="585" alt="Screenshot 2020-10-27 at 10 28 40 AM" src="https://user-images.githubusercontent.com/27020381/97258800-44ba5300-183f-11eb-8a1a-8d405cb27ad2.png">

Reply:
Yeah, it is true. Task should be the only element in this case. 👍